nuagekubemon/client/nuagevsdclient.go

```diff
@@ -885,6 +898,94 @@ func (nvsdc *NuageVsdClient) CreateEgressAclTemplate(domainID string) (string, e
 	}
 }
 
+func (nvsdc *NuageVsdClient) CreateIngressAclTemplateForNamespaceAnnotations(domainID string) (string, error) {
+	result := make([]api.VsdAclTemplate, 1)
+	payload := api.VsdAclTemplate{
+		Name:              api.ZoneAnnotationTemplateName,
+		DefaultAllowIP:    true,
+		DefaultAllowNonIP: true,
+		Active:            true,
+		Priority:          api.MAX_VSD_ACL_PRIORITY - 1,
```

Comment:
I have set the priority 1 unit higher than the priority for the default ACL template that we create.
https://github.com/nuagenetworks/nuage-kubernetes/blob/master/nuagekubemon/client/nuagevsdclient.go#L791

 Do you have a suggestion for changing the priority ?